src/coreclr/jit/importer.cpp

```diff
@@ -1042,30 +1030,30 @@ bool Compiler::impCheckImplicitArgumentCoercion(var_types sigType, var_types nod
         return true;
     }
 
-    if (TypeIs(sigType, TYP_BOOL, TYP_UBYTE, TYP_BYTE, TYP_USHORT, TYP_SHORT, TYP_UINT, TYP_INT))
```

Comment:
`TypeIs` matches the names we have elsewhere, like `OperIs`. I don't think its worth changing without doing considering a standard approach for the entire JIT.

Reply:
The problem is that `TypeIs` is declared for `GenTree` as a member function, so if we add a global `TypeIs` (we can't declare methods inside enum, even if it is a enum class) for each `GenTree:method() { TypeIs() }` we will deal with c++ overload resolution that will choose between `GenTree:TypeIs()` and global `::TypeIs()` that could lead to errors.